spec/unit/resources/logical_interconnect_group/load_resource_with_synergy_properties_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test_api300_synergy::logical_interconnect_group_load_resource_with_synergy_properties' do
```

Comment:
Please move this entire describe to the file `load_resource_with_properties_spec.rb`, and delete this file thereafter.